Documentation/project-docs/performance-tests.md

```diff
@@ -5,16 +5,89 @@ This document contains instructions for building, running, and adding Performanc
 
 Requirements
 --------------------
-
-To run performance tests, .NET portable v5.0 is required. This library is included in [the Visual Studio Community 2015 download](https://www.visualstudio.com/products/visual-studio-community-vs). To get the correct packages during installation, follow these steps after opening the installer:
+### Windows
+To run performance tests on Windows, .NET portable v5.0 is required. This library is included in [the Visual Studio Community 2015 download](https://www.visualstudio.com/products/visual-studio-community-vs). To get the correct packages during installation, follow these steps after opening the installer:
 1. Select "Custom Installation" if no installation is present, or "Modify" otherwise
 2. Check the "Universal Windows App Development Tools" box under the "Windows and Web Development" menu
 3. Install
+### Linux
+Performance tests on Linux require all of the same steps as they do for regular xunit tests - check out the linux instructions [here](https://github.com/dotnet/corefx/blob/master/Documentation/building/unix-instructions.md). Once you can have a directory on your Linux machine with a working corerun and xunit.console.netcore.exe (as well as the test dll containing your perf tests!), you only need to run the following command:
```

Comment:
Nit: I'd suggest changing "check out" to "see" or something like that... "check out" might be confusing with regards to a source control system.